backend/index.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const app = express();
+const mongoose = require("mongoose");
+const recipeEndpoints = require("./routes/recipe");
+
+require("dotenv").config();
+
+app.use(express.json());
+
+app.use((req, res, next) => {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  res.header('Access-Control-Allow-Methods', 'GET,POST,OPTIONS,DELETE,PUT');
+  next();
+});
+
+app.get("/", (req, res) => {
+  res.send("Hello world!");
+});
+
+mongoose
+  .connect(process.env.MONGO_URL)
```

Comment:
Hey Noah, so I'm looking through your website, I can get it to load up on my end but I can't see any of your recipe data on the site. looking through your routes and everything, it all looks good. The only thing that I see could be causing the issue is where you connect to your database here. I get the error "could not connect" from your catch statement. I am not familiar with the way you connected to it, does it work on your end?

Below is a link to a walkthrough of how to connect to your database, the last video in particular shows code example of connecting.
If this does not work, or something else is causing the issue I can try to help you find the issue, let me know if you have questions!

https://www.notion.so/Getting-started-with-MongoDB-Atlas-f31ad1f19409436486ddee8a103e2991

Reply:
Specifically, I don't see your mongo url anywhere, so it may just be that the file you stored it in did not upload

Reply:
Hello, yes sorry I did not include the connection URL as I included it in my .env as a security measure as I was not sure if it was safe to have a public MongoDB URL. I can send my .env file if that is a possible option. Sorry again, please let me know what works best for you. Thanks!

Reply:
that's alright, yeah so this is not best practice, usually a url like this would not be on git or would be better protected, but for simplicities sake of reviewing all these projects we do just need the URL in order to connect. But I would strongly suggest not using a real username and password that you would ever otherwise use, because this info is publicly available on github

Reply:
A bit more info here:
https://www.notion.so/Getting-started-with-MongoDB-Atlas-f31ad1f19409436486ddee8a103e2991
if you have other questions let me know :)

Reply:
Ok thanks, I just pushed with the URL in the file and it should work properly now. Please let me know if it does not as on my end everything works!